.mailmap

```diff
@@ -67,6 +67,8 @@ Andrey Kolomentsev <andrey.kolomentsev@docker.com> <andrey.kolomentsev@gmail.com
 André Martins  <aanm90@gmail.com> <martins@noironetworks.com>
 Andy Rothfusz <github@developersupport.net> <github@metaliveblog.com>
 Andy Smith <github@anarkystic.com>
+Andy Zhang <andy.zhangtao@hotmail.com>
+Andy Zhang <andy.zhangtao@hotmail.com> <ztao@tibco-support.com>
```

Comment:
Wondering why you added this one; as it looks like the hotmail address was neither used as actual author, nor as "signed-off-by"; was it because the `tibco-support.com` is no longer active?

```
git log --author=andy.zhangtao@hotmail.com
# no entries

git log --author=ztao@tibco-support.com
commit 098fdb6408e4e630c1c78b8283c4bc716cc46698
Author: andy <ztao@tibco-support.com>
Date:   Fri Jan 8 08:32:32 2016 +0800

(...)

    Signed-off-by: Andy Zhang <andy.zhangtao@hotmail.com>

(...)

    Signed-off-by: andy <ztao@tibco-support.com>
```

Reply:
It was because it was never used with a real name, and `tibco-support.com` no longer has MX records. We could use the name with the old email, but I think this is a bit simpler.